types/rlp_unmarshal.go

```diff
@@ -13,6 +13,8 @@ type RLPUnmarshaler interface {
 
 type unmarshalRLPFunc func(p *fastrlp.Parser, v *fastrlp.Value) error
 
+type unmarshalRLPFromCbFunc func(TxType, *fastrlp.Parser, *fastrlp.Value) error
```

Comment:
Nitpick (so that we are consistent with naming, since we have `unmarshalRLPFunc` from before). WDYT?

```suggestion
type unmarshalRLPFromFunc func(TxType, *fastrlp.Parser, *fastrlp.Value) error
```

Reply:
Done